src/Views/Form/Templates/Sequoia/assets/js/form.js

```diff
@@ -849,4 +853,36 @@
 			window.parentIFrame.sendMessage( { action: 'giveScrollIframeInToView' } );
 		}
 	}
+
+	/**
+	 * Update decimal donation levels amount
+	 *
+	 * @unreleased
+	 */
+	function updateFormDonationLevelsLabels() {
+		$( '.give-form' ).each( ( i, form ) => {
+			const donationForm = $( form );
+			const donationLevels = Give.form.fn.getVariablePrices( donationForm );
+
+			$.each( donationLevels, function( j, level ) {
+				if ( 'custom' === level.price_id ) {
+					return;
+				}
+
+				const symbol = Give.form.fn.getInfo( 'currency_symbol', donationForm );
+				const position = Give.form.fn.getInfo( 'currency_position', donationForm );
+				const precision = Give.form.fn.getInfo( 'number_decimals', donationForm );
```

Comment:
@alaca These three variables could be defined outside of the loop and re-used for reach iteration.

Reply:
@kjohnson you are right! 

Resolved 7c5a377c